API/Controllers/SchoolsController.cs

```diff
@@ -0,0 +1,132 @@
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using API.Models;
+using API.Data;
+using API.DTOs;
+
+namespace API.Controllers;
+
+[ApiController]
+[Route("/[controller]")]
+public class SchoolsController : ControllerBase
+{
+    private readonly myDbContext m_DbContext;
```

Comment:
I dislike naming here 
instead of m_DbContext I would just write myDbContext 
Also it is quite common in C# to use _ in the class variables 
meaning that it should be _myDbContext

Reply:
Fixed